lib/fluent/logger/level_fluent_logger.rb

```diff
@@ -0,0 +1,63 @@
+#
+# Fluent
+#
+# Copyright (C) 2011 FURUHASHI Sadayuki
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+require 'msgpack'
+require 'socket'
+require 'monitor'
+require 'logger'
+require 'json'
+require 'fluent/logger/core/base'
+require 'fluent/logger/core/fluent_logger_base'
+
+module Fluent
+  module Logger
+    class LevelFluentLogger < ::Logger
+      include Core::Base
+      include Core::FluentLoggerBase
```

Comment:
::Logger has `#reopen`, `#<<` `#close` public methods also.

Since `Core::FluentLoggerBase` has `#close` method, it is implemented.
But, `#reopen` is not implemented. It would be nice if it reconnect in the case of fluent-logger-ruby.
`#<<` is also not implemented. I feel it is fine to do nothing with this method in the case of fluent-logger-ruby.